src/tjota_db.erl

```diff
@@ -84,14 +85,63 @@ create_table(user) ->
         )
     ", [?KEYSPACE, ?TABLE_USER]));
 
-create_table(session) -> not_implemented;
-create_table(room) -> not_implemented;
-create_table(message) -> not_implemented;
+create_table(session) -> 
+    {ok, Client} = get_cqerl_client(),
+    {ok, _} = cqerl:run_query(Client, io_lib:format("
+        CREATE TABLE IF NOT EXISTS ~s.~s (
+        )
+    ", [?KEYSPACE, ?TABLE_SESSION]));
+
 
-create_table(user_room) -> not_implemented;
-create_table(room_user) -> not_implemented.
+create_table(room) -> 
+    {ok, Client} = get_cqerl_client(),
+    {ok, _} = cqerl:run_query(Client, io_lib:format("
+        CREATE TABLE IF NOT EXISTS ~s.~s (
+            room-name TEXT,
+            id UUID,
+            users TEXT,
+            type TEXT,
+            data BLOB,
+            PRIMARY KEY (id)
```

Comment:
The primary key is fine. As a good readability practice, in the create table, write the columns that belong to the primary key at the top.

```
CREATE TABLE ... (
    id TEXT, <-- pk at the top
    <<other columns here>>
    ...,
    PRIMARY KEY (id)
)
```